### PR TITLE
S5neo

### PR DESCRIPTION
**Pull request by Stricted**
compiles fine
need to test if the kernel boots

### CODE REVIEW

**Comment by Stricted** [comment]
some informations about the bcmdhd_mm change
if i leave it as it was ill get these compilation errors

```
  CC      drivers/net/wireless/bcmdhd/wl_roam.o
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c: In function 'dhd_pm_callback':
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:740:22: error: passing argument 1 of 'dhd_wakelock_waive' from incompatible pointer type [-Werror]
   dhd_wakelock_waive(dhdinfo);
                      ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:713:12: note: expected 'struct dhd_pub_t *' but argument is of type 'struct dhd_info_t *'
 static int dhd_wakelock_waive(dhd_pub_t *pub);
            ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:742:24: error: passing argument 1 of 'dhd_wakelock_restore' from incompatible pointer type [-Werror]
   dhd_wakelock_restore(dhdinfo);
                        ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:714:12: note: expected 'struct dhd_pub_t *' but argument is of type 'struct dhd_info_t *'
 static int dhd_wakelock_restore(dhd_pub_t *pub);
            ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c: At top level:
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:7550:5: error: conflicting types for 'dhd_wakelock_waive'
 int dhd_wakelock_waive(dhd_info_t *dhdinfo)
     ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:713:12: note: previous declaration of 'dhd_wakelock_waive' was here
 static int dhd_wakelock_waive(dhd_pub_t *pub);
            ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:7569:5: error: conflicting types for 'dhd_wakelock_restore'
 int dhd_wakelock_restore(dhd_info_t *dhdinfo)
     ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:714:12: note: previous declaration of 'dhd_wakelock_restore' was here
 static int dhd_wakelock_restore(dhd_pub_t *pub);
            ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:713:12: error: 'dhd_wakelock_waive' used but never defined [-Werror]
 static int dhd_wakelock_waive(dhd_pub_t *pub);
            ^
../../../../../../kernel/samsung/a3xelte/drivers/net/wireless/bcmdhd/dhd_linux.c:714:12: error: 'dhd_wakelock_restore' used but never defined [-Werror]
 static int dhd_wakelock_restore(dhd_pub_t *pub);
            ^
cc1: all warnings being treated as errors
/home/buildtest/android/system/kernel/samsung/a3xelte/scripts/Makefile.build:308: recipe for target 'drivers/net/wireless/bcmdhd/dhd_linux.o' failed
make[5]: *** [drivers/net/wireless/bcmdhd/dhd_linux.o] Error 1
make[5]: *** Waiting for unfinished jobs....
/home/buildtest/android/system/kernel/samsung/a3xelte/scripts/Makefile.build:455: recipe for target 'drivers/net/wireless/bcmdhd' failed
make[4]: *** [drivers/net/wireless/bcmdhd] Error 2
/home/buildtest/android/system/kernel/samsung/a3xelte/scripts/Makefile.build:455: recipe for target 'drivers/net/wireless' failed
make[3]: *** [drivers/net/wireless] Error 2
/home/buildtest/android/system/kernel/samsung/a3xelte/scripts/Makefile.build:455: recipe for target 'drivers/net' failed
make[2]: *** [drivers/net] Error 2
/home/buildtest/android/system/kernel/samsung/a3xelte/Makefile:813: recipe for target 'drivers' failed
make[1]: *** [drivers] Error 2
Makefile:130: recipe for target 'sub-make' failed
make: *** [sub-make] Error 2
make: Leaving directory '/home/buildtest/android/system/kernel/samsung/a3xelte'
ninja: build stopped: subcommand failed.
build/core/ninja.mk:151: recipe for target 'ninja_wrapper' failed
make: *** [ninja_wrapper] Error 1
make: Leaving directory '/home/buildtest/android/system'

#### make failed to build some targets (12 seconds) ####

buildtest@lineageos:~/android/system$

```

File: security/selinux/Makefile [hunk 1]
```diff
@@ -2,12 +2,6 @@
 # Makefile for building the SELinux module as part of the kernel tree.
 #
 
-ifeq ($(SEC_BUILD_OPTION_PRODUCT_SHIP), true)
-ifeq ($(SEC_BUILD_OPTION_SELINUX_ENFORCE),true)
-EXTRA_CFLAGS += -DCONFIG_ALWAYS_ENFORCE=true
-endif
-endif
-
```
Comment (aapav01) [inline security/selinux/Makefile]: no effect commit

Comment (Stricted) [reply security/selinux/Makefile]: removed it

**Comment by Stricted** [comment]
kernel boots fine
@aapav01  if you are ok with this i merge it

**Comment by Stricted** [comment]
i tried to use the a5 kernel too without any luck, the a5 kernel bootloops
so if youre going to make an common kernel use this one please

**Pull request merged**